pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -583,9 +584,10 @@ class Build {
             try {
                 fullVersionOutput = context.readFile(versionPath)
                 context.println "SUCCESS: version.txt found"
-            } catch (NoSuchFileException e) {
-                context.println "ERROR: ${versionPath} was not found. Exiting..."
-                throw new Exception()
+            } catch (Exception e) {
+                context.println "WARNING: ${versionPath} was not found. Likelihood is this is a riscv build which creates a different version file. Attempting to read ${riscvVersionPath}"
```

Comment:
This is a little bit of a red herring - can we detect if we are meant to be looking for riscv and predicate some of this debug accordingly?

Reply:
I figured asking for forgiveness was better than asking for permission. Nevertheless, it should be pretty easy to adapt to look for riscv